packages/boxel-ui/README.md

```diff
@@ -0,0 +1 @@
+Note: icon components in addon/src/icons are code-generated from svg files using `pnpm rebuild:icons` from the addon project.
```

Comment:
i think there is another rebuild script for usage right?

Reply:
I improved these docs.